app.rb

```diff
@@ -31,19 +32,74 @@
 config_overrides[:port] = ENV['APRUVE_PORT'] unless ENV['APRUVE_PORT'].nil?
 Apruve.configure(ENV['APRUVE_API_KEY'], apruve_environment, config_overrides)
 merchant_id = ENV['APRUVE_MERCHANT_ID']
+corporate_account_id = ENV['APRUVE_CORPORATE_ACCOUNT_ID']
 
 before '/webhook_notify' do
   request.body.rewind
   @webhook_body = request.body.read
 end
 
+get '/offline_order' do
+  erb :offline_orders
+end
+
+get '/customers' do
+  content_type :json
+  send_file File.expand_path('customers.json', settings.public_folder)
```

Comment:
~~With Michael and Julian being the primary users of demo (and hardcoding for a single merchant), we could use the API [endpoint for a specific merchant corporate account](https://docs.apruve.com/docs/versions/4.0.0/resources/merchants/endpoints/retrieve-a-merchant-corporate-account), in lieu of a static JSON file. There's an array field returned for [`authorized_buyers`](https://gist.github.com/dbelling/e869a022ca950fd71d5d06331c05c302#file-corporate_account-json-L8-L14) on the `CorporateAccount` object.~~

On second thought - our ruby demo doesn't currently support this endpoint - good call with the JSON file. I've made a [task to update the library](https://apruvit.atlassian.net/browse/AP-5533) for fetching this resource.